### ISSUE TITLE
accessoryRight of Input prop not working on expo andriod

### ISSUE THREAD

**Issue by Joshyahweh** [akveo/react-native-ui-kitten#1733]
## 🐛 Bug Report

When I tried to render an icon with accessoRight it gives an error on my expo go app that "Attempt to invoke virtual method 'int.java.lang.Integer.intValue()' on a null object reference"
It does not give the error on ios

## To Reproduce
`
export const PasswordInput: FC<Placeholder> = ({ placeholder, value, onChangeText, status }) => {
    const [secureTextEntry, setSecureTextEntry] = useState(true);
    const theme = useTheme();
    const toggleSecureEntry = () => {
        setSecureTextEntry(!secureTextEntry);
    };

    const renderIcon = (properties: IconProps) => (
        <TouchableWithoutFeedback onPress={toggleSecureEntry}>
            <Icon {...properties} name={secureTextEntry ? "eye-off" : "eye"} />
        </TouchableWithoutFeedback>
    );

    return (
        <Input
            placeholder={placeholder}
            accessoryRight={renderIcon}
            style={[
                InputStyles.password,
                {
                    borderColor: theme["color-primary-border"],
                    backgroundColor: theme["color-basic-transparent-100"],
                },
            ]}
            value={value}
            onChangeText={onChangeText}
            status={status}
            secureTextEntry={secureTextEntry}
        />
    );
};
`

## Expected behavior

You should get the error if you run it on expo andriod

**Comment by rigorcadiz** [comment]
I believe this is related to this issue. https://github.com/akveo/react-native-ui-kitten/issues/1675, I think it should have not been closed because the workaround is just a dirty patch.

TL;DR; Add `color=""` attribute in your Icon component.

**Comment by Joshyahweh** [comment]
Yes, you are correct.
